plugins/browser-plugin-media-tracking/src/buildMediaEvent.ts

```diff
@@ -56,7 +56,7 @@ function getHTMLMediaElementEntities(el: HTMLAudioElement | HTMLVideoElement): M
     readyState: READY_STATE[el.readyState] as MediaElement['readyState'],
     seekable: timeRangesToObjectArray(el.seekable),
     seeking: el.seeking,
-    src: dataUrlHandler(el.src),
+    src: dataUrlHandler(el.src || el.currentSrc),
```

Comment:
Consider: It appears that `src` is in fact already optional in the [schema](https://github.com/snowplow/iglu-central/blob/be84568d2dda1c272a15fb49cc63a916c180dd64/schemas/org.whatwg/media_element/jsonschema/1-0-0#L247) (it isn't in `required`), so I'm wondering if a null fallback is worth putting in here too, to prevent the empty string issue. I'm not quite sure if there's a situation where `src` and `currentSrc` will both be empty strings though.

Reply:
That's a great observation! 
In that case we indeed would want the schema to fail. IMO `src` should have been required so it could signify problematic cases.